### DISCUSSION TITLE
CoolProp - Ammonia & Water Mixtures

### DISCUSSION THREAD

**Discussion by astrotide** [CoolProp/CoolProp#2256] [Q&A]
I am trying to write a Python script to calculate enthalpy and entropy for an ammonia and water mixture using CoolProp/REFPROP. Below is what I have thus far:

import CoolProp.CoolProp as CP
CP.set_config_string(CP.ALTERNATIVE_REFPROP_PATH, "C:\Program Files (x86)\REFPROP\REFPRP64.DLL")
T1 = 323.0      # temperature [K]
P1 = 400000.0   # pressure [Pa]
x = 0.37        # mass fraction NH3 [-]
State = CP.AbstractState('REFPROP','Ammonia&Water')
State.set_mass_fractions([x, 1-x]) 
Rf = CP.generate_update_pair(CP.iT,T1,CP.iP,P1)
State.update(Rf[0],Rf[1],Rf[2])
h = State.hmass() 
s = State.smass() 

When I run this, I get the following error:

ValueError: fluid directories "FLUIDS" or "fluids" could not be found in the directory [C:\Program Files (x86)\REFPROP\REFPRP64.DLL]

I am new to CoolProp/REFPROP so any help or advise will be greatly appreciate!

Thanks

**Comment by ibell** [comment]
The provided path should be that to the Refprop folder,  not the DLL

**Reply by astrotide** [reply]
Thanks! I really appreciate your help.
